src/content/changes/highlights/2021-06-11.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2021-06-14T10:00:00
+title: Highlights for the week ending Jun 11, 2021
+changes_categories:
+- Highlights
+owner:
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+As we aim to align our terminology with the Kubernetes project where possible, we started replacing the term _master node_ with _control plane node_.
+
+## Managed apps
+
+[Loki v0.2.0](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.2.0/) comes with upstream version 2.2.1. Please note that there is a manual intervention required when upgrading to this version. Check the release info for details.
+
```

Comment:
@ubergesundheit Can you help me say something interesting/valuable about that release?